src/infra/azure/storage/queue/client.ts

```diff
@@ -0,0 +1,24 @@
+import { QueueServiceClient, QueueClient } from "@azure/storage-queue";
+
+import * as TE from "fp-ts/TaskEither";
+
+export const createQueueClient = (
```

Comment:
clients must be singletons shared by all functions, so what you realy want here is to export the instance directly (not a factory method)

see the `database` method into `database.ts`